packages/backend/server/src/base/error/def.ts

```diff
@@ -319,6 +320,10 @@ export const USER_FRIENDLY_ERRORS = {
     message: ({ status, body }) =>
       `Invalid callback code parameter, provider response status: ${status} and body: ${body}.`,
   },
+  invalid_oauth_callback_client_nonce: {
+    type: 'bad_request',
+    message: 'Invalid callback client_nonce parameter.',
```

Comment:
users have no idea what client nonce is
`Invalid auth state. You might start the auth progress from another device`

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/5c0ymolP9B7QStCsS1RP/79523b1c-63ba-47f0-a56e-a2d9c68055a9.png)